packages/mongo/test/DatabaseCollection.test.ts

```diff
@@ -4,14 +4,18 @@ import { MongoDbCollectionProvider, MongoDbConnection } from "../src";
 const connection = new MongoDbConnection(process.env.CONNECTION_STRING!);
 let database: MongoDbCollectionProvider;
 
+function randomString() {
+    return `x${Math.random().toString(36).substring(7)}`;
```

Comment:
Wieso gibt es jetzt einen x Präfix?

Reply:
Weil dbs in ferretdb nicht mit ner Zahl anfangen dürfen und bei dem rendom string gibt's das sonst.